wireapp/wire-android-reloaded
# feat(menu): close the context menu when the back button pressed [AR-1535]

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-1535" title="AR-1535" target="_blank"><img alt="Bug" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10803?size=medium" />AR-1535</a>  Device Back button on delete options menu closes conversation
  </td></table>
  <br />
 




----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues
[AR-1535]

### Causes (Optional)

When the user open a context menu, and press back button, the context menu should get closed first.

### Solutions

Handle the back button in context menu.


### Testing

#### Test Coverage (Optional)


#### How to Test

Open any context menu by long press on conversation, messages ..., then press back button, the context menu should get closed first.

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

[AR-1535]:https://wearezeta.atlassian.net/browse/AR-1535

[AR-1535]: https://wearezeta.atlassian.net/browse/AR-1535?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[AR-1535]: https://wearezeta.atlassian.net/browse/AR-1535?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
